pnpm-sync-api-demo/README.md

```diff
@@ -1,4 +1,22 @@
 # pnpm-sync-api-demo
+
 Here, we will show examples on how to use pnpm-sync APIs. The APIs mainly used where you want to integrate pnpm-sync into your own Monorepo management tools. 
 
-To be continued.
\ No newline at end of file
+## Folder structure
+
+We will use some test cases to demonstrate how to consume the APIs from `pnpm-sync-lib`.
+
+The main app to drive the test cases is under `pnpm-sync-api-demo` folder. All test cases are under `pnpm-sync-api-demo/src/test` folder.
+
+There are some sample apps and libs under `pnpm-sync-api-demo/text-fixtures` to be used by the test cases. 
+
+## Instructions
+
+### 1. Run `pnpm install` to install all dependencies
```

Comment:
Run `pnpm install` in which directory? It's unclear how these instructions relate to the top-level README.md for the monorepo:

https://github.com/tiktok/pnpm-sync/blob/cd263653f6e73c087f811dd7b4a41b9179c269e3/README.md?plain=1#L29-L49

Also, is it possible for these tests to run automatically as part of `pnpm --recursive run build` or `pnpm --recursive run test` from the top level?

Maybe the top-level README.md should explain how to build+test all projects, and then the instructions in this folder would explain how to build+test just this one project. (?)